src/mmw/js/src/compare/templates/compareScenarios.html

```diff
@@ -0,0 +1,7 @@
+<div id="edge-fade"></div><!-- Blur offcanvas scenario -->
+
+<table> <!-- Table allows for horizontal scrolling. -->
```

Comment:
Ah good point. I forget that not all platforms have invisible scrollbars like OS X and iOS. But at the same time I think that allowing native scroll in one platform but not in another is kind of limiting. Perhaps we can look in to tapping / clicking the edge scenario bringing it into view. I definitely think we should have more than one way of scrolling the list.